docs/basic-docs/atomicdex-api-legacy/order_status.md

```diff
@@ -12,12 +12,30 @@ The `order_status` method returns the data of the order with the selected `uuid`
 
 #### Response
 
-| Structure | Type   | Description                            |
-| --------- | ------ | -------------------------------------- |
-| type      | string | type of the order ("Maker" or "Taker") |
-| order     | object | order data                             |
-| base_orderbook_ticker            | string                     | the ticker of the base currency if `orderbook_ticker` is configured for the base currency in `coins` file. If not defined, will return a null value. |
-| rel_orderbook_ticker            | string                     | the ticker of the rel currency if `orderbook_ticker` is configured for the rel currency in `coins` file. If not defined, will return a null value. |
+| Structure                    | Type   | Description                            |
+| ---------------------------- | ------ | -------------------------------------- |
+| type                         | string | type of the order ("Maker" or "Taker") |
+| order                        | object | order data                             |
```

Comment:
Could you please add separate definitions of Taker and Maker orders? E.g. how it's done for `my_swap_status` for each event. To catch taker order status, you can create an unmatchable order and call the RPC. If it will be `FillOrKill`, you will also have an example of how it looks when canceled by timeout.

Reply:
@smk762 As of now, only maker order fields are listed in the table. Could you please add separate `TakerOrder` and `MakerOrder` fields descriptions? They differ significantly.